### ISSUE TITLE
Distribution as a docker image as well

### ISSUE THREAD

**Issue by IreshMM** [mattrobenolt/jinja2-cli#131]
This is a proposal to have a Dockerfile in the repository for an official container image for those who are looking to use jinja2-cli in a container. Of course, we can always have our own image created having jinja2-cli installed in it but I suppose having an official image with a well defined, easy to use interface for passing inputs and outputs to/from the container, to use it would be favorable for most people.

TL/DR: Adding a new release channel as a docker image.

**Comment by mortenlj** [comment]
While not "official" as in created by the same author, there is a docker image already: https://github.com/dinuta/jinja2docker
